System/RTEError.cpp

```diff
@@ -48,12 +48,15 @@ namespace RTE {
 
 	bool RTEAssertFunc(bool expression, const char *description, const char *file, int line, bool &alwaysIgnore) {
 		if (!expression) {
-			// TODO: Make this display a box in the game asking whether to ignore or abort")
-			// For now, always abort.
+			// TODO: Make this display a box in the game asking whether to ignore or abort. For now, always abort.
+
+#if defined DEBUG_BUILD || defined MIN_DEBUG_BUILD 	
 			char error[512];
-			sprintf_s(error, sizeof(error), "Assertion failed: %s", description);
+			sprintf_s(error, sizeof(error), "Assertion failed:\n\n%s", description);
```

Comment:
Hmm, what does description actually have/do here, is there any benefit to this really?

Reply:
:confused_jackie:
`description` is `RTEAssert(expression, "Literally the assert message")`.

Reply:
Sorry, wasn't clear. I'm wondering what the benefit of the `"Assertion failed:\n\n%s", description` stuff is vs just printing error. Like does this debug extra bit actually help any?

Reply:
I guess it doesn't.
I did this so the crash messages in final look cleaner since nobody really cares that an assertion failed, and we find those by the error message anyway.

Reply:
As discussed in discord, I think we can just get rid of that "Assertion failed" bit completely since it's never really looked at.